### PR TITLE
Update api-management-service.md

### PR DESCRIPTION
**Pull request by TCP1231**

### CODE REVIEW

File: changelogs/api-management-service.md [hunk 1]
```diff
@@ -4,15 +4,15 @@
 
 ### Highlights
 
-1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2. ([Docs](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview))
+-We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2. ([Docs](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview))
 
 ### New features, fixes and improvements
 
-1. We have added a feature flag for exporting API securitywhen exporting customer API to swagger.
+-Minor bug fixed and improvements.
```
Comment (vladvino) [inline changelogs/api-management-service.md:11]: Please correct spelling: bug fixe**s**

File: changelogs/api-management-service.md [hunk 2]
```diff
@@ -4,15 +4,15 @@
 
 ### Highlights
 
-1. We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2. ([Docs](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview))
+-We're thrilled to announce the public preview launch of our latest Azure API Management pricing tiers: Basic v2 and Standard v2. ([Docs](https://learn.microsoft.com/azure/api-management/v2-service-tiers-overview))
```
Comment (vladvino) [inline changelogs/api-management-service.md:7]: Please add a space between "-" and the text in all items

**Review by vladvino — CHANGES_REQUESTED** [review]

**Review by TCP1231 — COMMENTED** [review]
completed

**Review by TCP1231 — COMMENTED** [review]
completed changes

**Pull request merged**